.changeset/short-vans-unite.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/button': patch
+---
+
+Ensures that `children` is not a required prop on component
```

Comment:
```suggestion
Fix type signature to not have `children` be a required prop
```

Reply:
Accidentally merged too soon but will make a branch shortly to fix this